opencbm/cbmcopy/main.c

```diff
@@ -488,8 +490,9 @@ int ARCH_MAINDECL main(int argc, char **argv)
                         {
                             if(output_name)
                             {
-                                strncpy(buf, output_name, 16);
-                                buf[16] = '\0';
+                                /* leave room for file type, mode and '\0'  */
+                                strncpy(buf, output_name, FN_BUFFER_SIZE-5);
```

Comment:
Why don't you use "sizeof buf" instead of "FN_BUFFER_SIZE"?

Reply:
Because the code didn't use a `sizeof` before, either :smiley: 

But sure, I changed it and pushed the new version.